build/build-release.sh

```diff
@@ -46,6 +46,7 @@ build_release() {
 }
 
 clone_repo() {
+    git config --system --add safe.directory '*'
```

Comment:
We could disable the check for a specify directory by moving this command in the Makefile similar to what OPA does.